arangod/Indexes/IndexIterator.h

```diff
@@ -63,17 +63,52 @@ class Methods;
 
 struct IndexIteratorOptions;
 
+class IndexIteratorCoveringData {
+ public:
+  virtual ~IndexIteratorCoveringData() = default;
+  virtual VPackSlice at(size_t i) = 0;
+  virtual bool isArray() const noexcept = 0;
+  virtual VPackSlice value() const {
```

Comment:
`virtual VPackSlice value() const = 0;`?

Reply:
Some implementation do not need "value" so I wanted to implement "assert only" function just once